database/user_test.go

```diff
@@ -30,12 +30,12 @@ func TestStorageUsed(t *testing.T) {
 		size   int64
 		result int64
 	}{
-		{size: 0, result: 4 * MiB},
-		{size: 1 * MiB, result: 4 * MiB},
-		{size: 4 * MiB, result: 4 * MiB},
-		{size: 5 * MiB, result: (4 + 40) * MiB},
-		{size: 50 * MiB, result: (4 + 2*40) * MiB},
-		{size: 500 * MiB, result: (4 + 13*40) * MiB},
+		{size: 0, result: SizeBaseSector},
+		{size: 1 * MiB, result: SizeBaseSector},
+		{size: 4 * MiB, result: SizeBaseSector},
+		{size: 5 * MiB, result: SizeBaseSector},
```

Comment:
Looks like this is different now (before it was 44 mib). Also, why doesn't this repo have CI to run tests automatically?

Reply:
Fat fingers. Fixed.

CI is a good point, added an issue: https://github.com/NebulousLabs/skynet-accounts/issues/35